docs/one-wire-devices.md

```diff
@@ -0,0 +1,157 @@
+
+:::
+
+:::warning
+
+Some One-Wire devices will require a pull-up resistor on the Data line (DQ). For example, the popular DS18B20 temperature sensor requires a 4.7 kΩ pull-up resistor on the DQ to operate properly.
+
+:::
+
+
+### Stage 2: Connect Power & Ground
+
+Connect the power and ground wires belonging to your One-Wire device to the corresponding power and ground pins on the Omega2 board. 
+
+### Stage 3: Connect the DQ
+
+Connect the DQ from your One-Wire device to the GPIO19 pin on the Omega2 board. 
+
+## Enable One-Wire Device Support
+
+
+Once the physical connections are made, install dts overlay package to register a one-wire master in Linux associated with GPIO19 to communicate with the one-wire slave device.  
+
+**Step 1:**  Install the `onion-dt-overlay-w1-gpio` package using opkg
+
+```js
+opkg install onion-dt-overlay-w1-gpio
+```
+**Step 2:** After the package is successfully installed, GPIO19 will act as a One-Wire Master. 
+
+```js
+/sys/devices/w1_bus_master1
+```
+<h2 id="find">Find Connected One-Wire Devices</h2>
+
+**Step 1:** Navigate to the relevant directory where One-Wire devices are listed. This is under `/sys/devices/w1_bus_master1`
+
+
+```js
+cd /sys/devices/w1_bus_master1
+ls
+```
+
+**Step 2:** Perform a check to find if slave devices are connected or not. 
+
+```js
+cat /sys/devices/w1_bus_master1/w1_master_slave_count
+```
+
+**Step 3:** The command in step 2 will return the count of the number of slave devices connected. 
+
+| Result            | Interpretation                            |
+|-------------------|-------------------------------------------|
+| 1                 | A slave device is connected               |
+| 0                 | A slave device is connected               |
+
+:::note
+
+The One-Wire bus master kernel module scans the data pin every 10 seconds for new devices connected, if any, hence, it is recommended to wait a little while and try again. 
+
+:::
+
+**Step 4:** If a slave device is connected, run the following command: 
+
+```js
+cat/sys/devices/w1_bus_master1/w1_master_slaves
+```
+
+This will return a (newline delimited) list of your slave device ID(s). The Device ID is the serial number of your One-Wire device. Example of a device ID: 28-000123456789. 
+
+:::note
+
+Each device will have a different serial number, making it complex to use One-Wire devices programmatically. 
+
+:::
+
+## Read Data from a Connected One-Wire Device
+
+**Step 1:** Navigate to the specific slave device directory and access the data from the __w1_slave file__ by running the following command: 
+
+```js
+cat /sys/devices/w1_bus_master1/DeviceID/w1_slave
+```
+
+The `<DeviceID>` in the command above is the serial number of your One-Wire device. Please refer to **Step 4** of the command section "Find Connected One-Wire Devices" to learn more about finding the "DeviceID". 
```

Comment:
And here, since we asked them to take note of the device id, we can say something like "The `<DeviceID>` in the command above is the serial number of your One-Wire device we noted in the previous section"

We should still keep the reference to step 4 for completeness